Apps/NeuroscanAcquireLSL/nsatransporter.cpp

```diff
@@ -0,0 +1,113 @@
+#include "nsatransporter.h"
+
+NsaTransporter::NsaTransporter() {}
+
+void NsaTransporter::init() {
+    npackets = 0;
+    socket = new QTcpSocket;
```

Comment:
This will leak memory because socket is newer deleted. Maybe change socket to a `std::unique_ptr<QTcpSocket>`?

Reply:
I think it's better to call `socket->deleteLater()` in `~NsaTransporter()`.